src/mono/wasm/build/WasmApp.targets

```diff
@@ -49,6 +49,8 @@
       - $(EmccFlags)                        - Emcc flags used for both compiling native files, and linking
       - $(EmccExtraLDFlags)                 - Extra emcc flags for linking
       - $(EmccExtraCFlags)                  - Extra emcc flags for compiling native files
+      - $(EmccAssertions)                   - Corresponds to `ASSERTIONS` arg for emcc. Default false.
```

Comment:
If we want to support only `-sASSERTIONS=1` via `EmccAssertions: bool`, then it would be better to rename this to `EmccEnableAssertions`.

Reply:
I just noticed there is also `_EmccAssertionLevelDefault`